scene/animation/animation_blend_tree.cpp

```diff
@@ -34,6 +34,7 @@
 
 void AnimationNodeAnimation::set_animation(const StringName &p_name) {
 	animation = p_name;
+	emit_signal(SNAME("tree_changed"));
```

Comment:
The `tree_changed` is an exaggeration. It is a signal to use when the number of nodes in the tree increases or decreases, so you should fix it using available local functions such as `_validate_property()` instead.